src/components/search/index.tsx

```diff
@@ -1,35 +1,34 @@
 import Downshift from 'downshift';
-import { useState } from 'react';
+import { ReactNode, useState } from 'react';
 import { useAutocomplete } from '@atb/page-modules/departures/client';
 import style from './search.module.css';
 import VenueIcon from '@atb/components/venue-icon';
 import { andIf } from '@atb/utils/css';
-import GeolocationButton from '@atb/components/search/geolocation-button';
 import { GeocoderFeature } from '@atb/page-modules/departures';
-import SwapButton from '@atb/components/search//swap-button';
 
 type SearchProps = {
   label: string;
   onChange: (selection: any) => void;
-  initialQuery?: string;
-  onSwap?: () => void;
+  button?: ReactNode;
+  initialFeature?: GeocoderFeature;
 };
 
 export default function Search({
   label,
   onChange,
-  initialQuery = '',
-  onSwap,
+  button,
+  initialFeature,
 }: SearchProps) {
-  const [query, setQuery] = useState(initialQuery);
+  const [query, setQuery] = useState('');
   const { data } = useAutocomplete(query);
 
   return (
     <Downshift<GeocoderFeature>
-      initialInputValue={query}
+      initialSelectedItem={initialFeature}
+      initialInputValue={geocoderFeatureToString(initialFeature)}
```

Comment:
Is this needed in addition to the `itemToString`?

Reply:
No 😄